scrapers/Algolia_Adultime.yml

```diff
@@ -2,33 +2,51 @@ name: "Adultime"
 sceneByURL:
   - action: script
     url:
+      - accidentalgangbang.com/en/video/
+      - adamandevepictures.com/en/video/
       - adulttime.com/en/video/
+      - AdultTimePilots.com/en/video/
       - analteenangels.com/en/video/
       - assholefever.com/en/video/
+      - BeingTrans247.com/en/video/
```

Comment:
```suggestion
      - beingtrans247.com/en/video/
```

Reply:
as above, I could add the lowercase variant, but would want to keep the TitleCase domain as that's what Algolia API returns

Reply:
I've added the lowercase variant

Reply:
PascalCase variant removed